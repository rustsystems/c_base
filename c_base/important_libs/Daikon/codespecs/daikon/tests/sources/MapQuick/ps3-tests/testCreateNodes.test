#create the nodes we'll use
CreateNode n1 2
CreateNode n2 20
CreateNode n3 4
CreateNode n4 4
CreateNode n5 12
CreateNode n6 1
CreateNode n7 3
CreateNode n8 9
CreateNode n9 5
CreateNode n10 3
CreateNode n11 11
