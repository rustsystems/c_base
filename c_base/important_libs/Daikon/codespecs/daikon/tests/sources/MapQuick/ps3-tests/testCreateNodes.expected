#create the nodes we'll use
created node n1 with cost 2
created node n2 with cost 20
created node n3 with cost 4
created node n4 with cost 4
created node n5 with cost 12
created node n6 with cost 1
created node n7 with cost 3
created node n8 with cost 9
created node n9 with cost 5
created node n10 with cost 3
created node n11 with cost 11
